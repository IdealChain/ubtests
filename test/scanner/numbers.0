010
110
1a
1f
0
&10
&01
&00
