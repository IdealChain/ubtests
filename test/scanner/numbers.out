num 16
num 272
num 26
num 31
num 0
num 10
num 1
num 0
